src/components/Main/Main.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Main.css';
+import Zodiac from '../Zodiac/Zodiac';
+
+import { zodiac } from '../../data';
+
+export default function Main() {
+  return <main className='main'>
```

Comment:
Great work Alex! In our <main> tag is where you could add a background image for +2 points.